apps/web/src/pages/templates/components/SmsBubble.tsx

```diff
@@ -0,0 +1,159 @@
+import styled from '@emotion/styled';
+import { Skeleton } from '@mantine/core';
+import { colors, PencilOutlined } from '@novu/design-system';
+import { MouseEventHandler, useState } from 'react';
+
+const SmsBubbleHolder = styled.div`
+  position: relative;
+`;
+
+const SmsBubbleContainer = styled.div<{ isBlur: boolean; isError: boolean }>`
+  position: relative;
+  width: fit-content;
+  min-width: 160px;
+  min-height: 36px;
+  padding: 8px 12px;
+  border-radius: 20px;
+  background: ${({ isError }) => (isError ? colors.error : '#51ba52')};
```

Comment:
`#51ba52` this is the light green color unfortunately not from the color theme we have

Reply:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** Could we please check with Nik to make sure this will be part of our next color palette then? (I think we already have the base palette defined somewhere)‏